bitnami/redis/Chart.yaml

```
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
-version: 16.9.5
+version: 16.9.6
```

As this is a new functionality, we should bump the version to a new minor:
```suggestion
version: 16.10.0
```